api-module/src/main/java/org/devridge/api/domain/community/service/ProjectService.java

```diff
@@ -26,25 +34,43 @@ public class ProjectService {
     private final ProjectMapper projectMapper;
     private final MemberRepository memberRepository;
     private final S3Service s3Service;
+    private final ProjectQuerydslRepository projectQuerydslRepository;
+    private final ProjectSkillRepository projectSkillRepository;
+    private final SkillRepository skillRepository;
 
+    @Transactional
     public Long createProject(ProjectRequest request) {
         Long accessMemberId = SecurityContextHolderUtil.getMemberId();
         Member member = getMemberById(accessMemberId);
 
         Project project = projectMapper.toProject(request, member);
-        return projectRepository.save(project).getId();
+        Long projectId = projectRepository.save(project).getId();
+        createProjectSkill(request.getSkillIds(), projectId);
+
+        return projectId;
     }
 
     @Transactional
     public ProjectDetailResponse getProjectDetail(Long projectId) {
         Project project = getProjectById(projectId);
+
+        List<ProjectSkill> temp = project.getProjectSkills();
+        List<String> skills = new ArrayList<>();
+        for (ProjectSkill t : temp) {
```

Comment:
💊 temp라는 이름 보다는 명확한 이름을 써 주시면 좋을 것 같네요
```suggestion
        for (ProjectSkill skill : project.getProjectSkills()) {
```